.travis.yml

```diff
@@ -11,8 +11,14 @@ matrix:
       env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable' SYMFONY_DEPRECATIONS_HELPER=weak
     - php: 5.6
       env: DEPENDENCIES=dev
+    - php: 5.6
+      env: DEPENDENCIES='dev' SYMFONY_VERSION=2.8.*@dev
```

Comment:
we don't need both 2.8 and 3.0 here: one of them is installed by the normal ``DEPENDENCIES=dev`` version